api_docs/changelog.md

```diff
@@ -20,6 +20,11 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 7.0
 
+**Feature level 170**
+
+* [`POST /user_topics`](/api/update-user-topic):
+  Added a new endpoint to update the visibility policy of a topic.
```

Comment:
Technically, this endpoint updates the user's preferences for a topic -- the distinction is important in that in the future we may have other topic properties than just visibility policy managed ny this endpoint.

Reply:
Earlier I thought of documenting only those things which we currently offer and update the docs later on when we start supporting other things.

But the suggested text makes more sense. (It explains the future scope and the current features we offer.)